rules/core/block-no-empty/rule.js

```diff
@@ -0,0 +1,7 @@
+const _THROW = require('../../../modules/throwables');
+
+module.exports = {
+    rules: {
+        'block-no-empty': [true, _THROW.ERROR],
```

Comment:
@netsells/frontend thinking about it i think this rule should be a warning rather than an error. thoughts?

Reply:
I think 99% of the style rules should be warning. With the exception of `!important`

Reply:
I agree, most should be warnings.